### PR TITLE
Added --binary option to patch only for windows

### PR DESCRIPTION
**Pull request by marcomd**
#203
Let me know if there is something to fix

### CODE REVIEW

File: libv8.gemspec [hunk 1]
```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
     `git ls-files`.split("\n").reject {|f| f =~ /^out/}.map {|f| "vendor/v8/#{f}"}
   end
   s.files += Dir['vendor/v8/build/**/*']
+  s.files += Dir['vendor/gyp/**/*']
```
Comment (ignisf) [inline libv8.gemspec]: Why did you need to add this?

Comment (marcomd) [inline libv8.gemspec]: Without that line I could not compile but i don't remember if it was due to folder presence or to what it contains

**Comment by T-Boss** [comment]
Any word on when this will make it in a release? A project I'm working on is currently blocked by this.

**Comment by marcomd** [comment]
This version does not pass travis tests, i even removed all changes
